packages/serializer/src/utils/data-construction.util.ts

```diff
@@ -74,6 +74,15 @@ export const constructData = async (data: any, customConfig: ISerializerConfig =
 
   const config = { ...defaultConfig, ...customConfig };
 
+  // Allows serialization to be skipped
+  // Should only be used during development or pre-migration to a serializer
+  if(options.skipSerialization) {
+    if(config.attributes.length > 0) {
+      console.debug('Skipping serialization of attributes although defined (uses skipSerialization option)', config.attributes);
+    }
+    return data;
+  }
+
```

Comment:
@knor-el-snor instead of going through `constructData`, could we avoid it and just return `data` in `serialize` method? 

```diff
  async serialize<T = any>(data: any, dataSetConfig: IMeta = {}): Promise<ISerializedResponse<T | T[]>> {
    return {
      meta: constructMeta(data, this.resource, dataSetConfig),
-      data: await constructData<T>(data, this.config, this.options),
+      data,
    };
  }
```

I think it would be more obvious this way that `data` is not changing. We probably want to modify `meta` still and add `type: unchanged` or something similar

Reply:
Mmm we could, but it's actually making the code a lot uglier on the serializer side, especially if you wish to add the `type: unchanged` which then has to be all send from the parent function if we want to keep it abstracted. Will check

Reply:
@alenkis Updated